static/js/admin/vue-permissions-editor/tests/unit/staffRoles.spec.js

```diff
@@ -121,6 +129,59 @@ describe('staffRoles.vue', () => {
             done();
         });
     });
+    
+    it("displays names of containers that roles are assigned to in inherited table", (done) => {
+        wrapper = shallowMount(staffRoles, {
```

Comment:
Instead of overriding the whole mounted wrapper you should be able to do this `wrapper.setProps({ my props here });`
This should just override the props you want to update/change while inheriting the rest of the beforeEach() wrapper setup.